src/main/java/reposense/parser/ArgsParser.java

```diff
@@ -45,7 +53,7 @@ private static ArgumentParser getArgumentParser() {
         parser.addArgument("-output")
                 .metavar("PATH")
                 .type(Arguments.fileType().verifyExists().verifyIsDirectory().verifyCanWrite())
-                .setDefault(new File("."))
+                .setDefault(new File(generateReportName()))
                 .help("The path to the dashboard generated. "
                         + "If not provided, it will be generated in the current directory.");
```

Comment:
May need to change the help message for these too, mention that we will create a directory with the timestamp as name in the current directory.
`The path to the dashboard generated.` also sounds abit weird, maybe `The directory to output the generated dashboard.` sounds better?